docker-entrypoint.sh

```diff
@@ -47,6 +56,14 @@ if [[ "$MONGODB_DATABASE" ]]; then
    JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .queuePopulator.mongo.database=\"$MONGODB_DATABASE\""
 fi
 
+if [[ "$S3_HOST" ]]; then
+    JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .s3.host=\"$S3_HOST\""
```

Comment:
My understanding is that to allow extensions to access an S3 endpoint they need those two environment variables, which get translated into config.json. We introduced this `s3` config object for lifecycle. Is there any existing environment that is set automatically? I think NicolasT referred to kube setting some env var to other services automatically in another PR which could be used here.

Reply:
All kubernetes pods get injected with environment variables that have the updated IP and port information for all available services which I believe is what NicolasT was referring to and could potentially be used 🙂

Reply:
Wouldn't a simple `docker run` impose the weird var name format? It seems fine as an intermediary mapping step to inject say `CACHE_SERVICE_HOST` into `REDIS_HOST` (fictional names) in the pod template, but a name like `CACHE_SERVICE_HOST` is kind of dictated by the service names defined at runtime outside of the pod, whereas these variables in the entrypoint are defined at build time.

Reply:
Yeah exactly, they would be dictated by the potentially different service names and release names so we wouldn't be able to hard code the values. My point really being that I think the idea in general to be good, it would require to rework the entire entrypoint script (at the very least) to take something like release name as a parameter and assume the default services names for Zenko K8s specific context which seems like a lot of work for not really much benefit.